### ISSUE TITLE
Add possibility to add a manual admin user to cleaner_delete_users

### ISSUE THREAD

**Issue by abias** [catalyst/moodle-local_datacleaner#63]
Hi,

This feature request is targeted at the latest version of the plugin published on https://moodle.org/plugins/pluginversion.php?id=12918 and may be superseded by recent commits here in Github.

After (optionally) deleting really all users from one of our washing box instances, we would like to use local_datacleaner to add a manual (=manually authenticated) admin user who can access the cleaned Moodle instance to do his stuff. The credentials of this admin user should be configurable within the local_datacleaner settings. 

This is requested because 
* there may be users who should be able to play with a cleaned Moodle instance as admins, but don't have admin permissions in production.
* there may be reasons not to re-use the existing accounts of these users, for example because you un-hooked the cleaned Moodle instance from LDAP.

Thanks,
Alex

**Comment by brendanheywood** [comment]
Yes also seems useful, but I'd maybe approach it slightly differently.

If you do a fresh install of moodle there is always 2 users, id=1 is the universal guest user, and id=2 is the 'special' primary site admin. I think it would be better if these were never deleted but just reset to some clean state.

I am really curious whether all these requirements are used in combination? For instance if you are removing all the users (this issue) as well as all the courses (#60) then there really isn't much data left besides config. I want to make sure I haven't missed something in the bigger picture.

**Comment by abias** [comment]
> If you do a fresh install of moodle there is always 2 users, id=1 is the universal guest user, and id=2 is the 'special' primary site admin. I think it would be better if these were never deleted but just reset to some clean state.

I think your proposal makes sense :)

> I am really curious whether all these requirements are used in combination? For instance if you are removing all the users (this issue) as well as all the courses (#60) then there really isn't much data left besides config. I want to make sure I haven't missed something in the bigger picture.

I have tried to explain our user story in #60.
